cli/cmd/plugin/unmanaged-cluster/log/output.go

```diff
@@ -0,0 +1,231 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
@stmcginnis can you put this in `unmanaged-cluster/internal/hack`. I want to ensure:

1. The package name really stands out as a workaround.
2. The package is not importable and APIs/datatypes/public functions cannot be coded against.

This will add an extra import in the `cmd` package, but this is a good thing IMO.

Let me know if this is not possible.